apps/sms/index.html

```diff
@@ -127,8 +128,8 @@
   </head>
 
   <body role="application" class="skin-comms theme-communications">
-    <article id="main-wrapper" class="wrapper" data-position='right'>
```

Comment:
Yeah, I've already tried to start removing it several times - but always end up with out-of-scope changes that I don't want to include into PR :) Likely need separate bug for this one